sprocs/instance_question_select_manual_grading_objects.sql

```diff
@@ -2,7 +2,16 @@
 DROP FUNCTION IF EXISTS instance_question_select_manual_grading_objects(bigint);
 
 -- Retrieves the last variant for an instance question and last submission for the variant.
-
+-- TODO: Move this query to grab the most recent manual grading job for a given submission?
+--       Should we add the variant_id onto the grading_jobs table, and then query the variant ID
+--          off of that?   
+--       Should we be only presenting a question if it doesn't have an outstanding external grading
+--          job, or should we bubble up completion messages for the job?
+--       No, we should be basing this off of the submission. We should find the most recent submission
+--          for the given question (as we do), but instead of doing fun queries, we can do it based upon
+--          the most recently submitted manual grading job.
+--       We should let someone submit a manual grade, but we should warn them if an external job is hanging.
```

Comment:
- This query can quite quickly be modified to include grading jobs without any issues.
- Warning message if the external grader is hanging. We may have to learn about the external grader and how it fails to manage this. Don't know if we should include it in this or find a simpler way to work around this issue for now to avoid the added complexity.
- In the new logic, we always look for the last submission for an instance question and gather the necessary dependency objects based off of that, so it looks like some of this work has been done for you. We might need to merge our branches soon.